pom.xml

```diff
@@ -42,7 +42,8 @@
     </developers>
 
     <properties>
-        <gridsuite-dependencies.version>37.0.0</gridsuite-dependencies.version>
```

Comment:
to remove the gridsuite-dependencies dependency as network modification is part of gridsuite-dependencies